namespaces/kubernetes.yml

```diff
@@ -57,6 +68,32 @@ namespace:
       Each label name is a subfield of labels field.
       Each label name is de-dotted: dots in the name are replaced with
       underscores.
+    fields:
+      - name: deployment
+        type: string
+        example: logging-kibana-3
+        description: >
+          The deployment associated with this Kubernetes object
+      - name: deploymentconfig
+        type: string
+        example: logging-kibana
+        description: >
+          The deploymentconfig associated with this Kubernetes object
+        fields:
+          - name: raw
```

Comment:
I don't think you need raw here, as `deploymentconfig` itself should be not_analyzed